packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -1036,7 +1036,8 @@ class PubspecDependency extends PubspecLine {
       // We're a git: dependency. Return false so we'll be forgotten.
       return false;
     } else {
-      throw 'Could not parse additional details for dependency $name; line was: "$line"';
+      // We're an empty target.
```

Comment:
is there a way to confirm that the line describes empty target and keep `else { throw` for futher unexpected cases?

Reply:
Removed this code, instead I should use the regular yaml parser - all I need are the the list of dependencies.